outputs.tf

```diff
@@ -1,14 +1,49 @@
-output "id" {
-  description = "ID of the created example"
-  value       = module.this.enabled ? module.this.id : null
+output "security_group_ids" {
+  description = "IDs of the MWAA Security Group(s)"
+  value       = var.create_security_group ? concat(var.associated_security_group_ids, [module.mwaa_security_group.id]) : var.associated_security_group_ids
 }
 
-output "example" {
-  description = "Example output"
-  value       = module.this.enabled ? local.example : null
+output "s3_bucket_arn" {
+  description = "ARN of the S3 bucket"
+  value       = var.create_s3_bucket ? module.mwaa_s3_bucket.bucket_arn : var.source_bucket_arn
```

Comment:
use local var for this expression to make it DRY since the same expressions is used in the module

Reply:
Added